test/@yoda/barbapapa/index.test.js

```diff
@@ -0,0 +1,21 @@
+var test = require('tape')
+var Barbapapa = require('@yoda/barbapapa')
+
+test('should resolve dependencies', t => {
+  t.plan(3)
+  class Foo {
+
+  }
+  Foo.dependencies = [ 'bar' ]
```

Comment:
We have supported the `class-static` member:

```js
class Foo {
  static get dependencies () {
    return [ 'bar' ]
  }
  constructor (arg1) {
    // TODO
  }
}
```

Reply:
static getter looks not promising to me. Would preferring static property if possible in the case.

Reply:
Static property seems better choice to me, too :)